game/src/main/java/org/apollo/game/model/entity/Entity.java

```diff
@@ -34,6 +39,20 @@ public Entity(World world, Position position) {
 	@Override
 	public abstract boolean equals(Object obj);
 
+	/**
+	 * Gets the {@link EntityBounds} for this Entity.
+	 *
+	 * @return The EntityBounds.
+	 */
+	public EntityBounds getBounds() {
+
+		if(bounds == null) {
```

Comment:
There was a noticeable amount of overhead on startup without this. We also might not ever use this with certain entities like `Projectile`.

Reply:
I think this a problem with how map loading is implemented, but we can tackle it later.